menustyle.css

```diff
@@ -0,0 +1,48 @@
+* {
+    box-sizing: border-box;
+    font-family: 'Rock Salt', cursive;
+}
+header {
+    padding: 50px;
+}
+body {
+    background-image: url("trump.jpg");
+    background-size: cover;
+}
+h1 {
+    font-size: 80px;
+}
+header,
+.startBtn,
+h3 {
+    text-align: center;
```

Comment:
Seems like all your font styles are set to text-align: center on this page? Think of a way to set this style value so it cascades down the page and you don't have to repeat this line of code three times!